client/me/account-close/main.jsx

```diff
@@ -0,0 +1,114 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import page from 'page';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFigure from 'my-sites/site-settings/action-panel/figure';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import Button from 'components/button';
+
+class AccountSettingsClose extends Component {
+	goBack = () => {
+		page( '/me/account' );
+	};
+
+	handleDeleteClick = event => {
+		event.preventDefault();
+	};
+
+	render() {
+		const { translate } = this.props;
+		const hasAtomicSites = false;
+
+		return (
+			<div className="account-close" role="main">
+				<HeaderCake onClick={ this.goBack }>
+					<h1>{ translate( 'Close account' ) }</h1>
+				</HeaderCake>
+				<ActionPanel>
+					<ActionPanelTitle>{ translate( 'Close account' ) }</ActionPanelTitle>
+					<ActionPanelBody>
+						<ActionPanelFigure>
+							<h3>{ translate( 'These items will be deleted' ) }</h3>
+							<ul>
+								<li>{ translate( 'Posts' ) }</li>
+								<li>{ translate( 'Pages' ) }</li>
+								<li>{ translate( 'Media' ) }</li>
+								<li>{ translate( 'Users & Authors' ) }</li>
+								<li>{ translate( 'Domains' ) }</li>
+								<li>{ translate( 'Purchased Upgrades' ) }</li>
+							</ul>
+						</ActionPanelFigure>
+						{ ! hasAtomicSites && (
+							<div>
+								<p>
+									{ translate(
+										'Deletion {{strong}}can not{{/strong}} be undone, ' +
+											'and will remove all content, contributors and domains from your account.',
+										{
+											components: {
+												strong: <strong />,
+											},
+										}
+									) }
+								</p>
+								<p>
+									{ translate(
+										"If you're unsure about what deletion means or have any other questions, " +
+											'please chat with someone from our support team before proceeding.'
+									) }
+								</p>
+								<p>
+									<a href="/help/contact">{ translate( 'Contact support' ) }</a>
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Contact Support' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/me/account-close-confirm-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).